src/install/config/python.yaml

```diff
@@ -0,0 +1,750 @@
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-tornado-6-web-framework
+
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado-3
+  - /node/1971
+  - /docs/python/python-agent-and-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/nr-only-python-agent-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado-experimental
+  - /docs/agents/python-agent/hosting-mechanisms/introductory-tornado-4-support
+  - /docs/agents/python-agent/web-frameworks-servers/using-trace-decorators-tornado-coroutines
+  - /docs/agents/python-agent/trace-decorators-tornado-coroutines
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-tornado-6-web-framework
+
+  - /docs/agents/python-agent/web-frameworks-servers/uvicorn
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-uvicorn-web-server
+  - /docs/agents/python-agent/web-frameworks-servers/uvicorn-web-server
+  - /docs/apm/agents/python-agent/web-frameworks-servers/uvicorn
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-uwsgi-web-server
+  - /docs/python/python-agent-and-uwsgi
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-uwsgi
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-uwsgi
+  - /docs/apm/agents/python-agent/web-frameworks-servers
+  - /docs/agents/python-agent/web-frameworks-and-servers
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-uwsgi-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-waitress-web-server
+  - /docs/python/python-agent-and-waitress
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-waitress
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-waitress
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-waitress-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
+  - /docs/python/python-agent-and-web2py
+  - /docs/agents/python-agent/web-frameworks/python-agent-and-web2py
+  - /docs/agents/python-agent/web-frameworks/python-agent-web2py
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
+
+
+# -----------------------------------------------------------------
+# Define the categories for each type of question you'll be asking:
+# -----------------------------------------------------------------
+appInfo:
+
+
+
+  - optionType: python-non-web
+    label: "We'll show you how to instrument your non-web Python app"
+    placeholder: 'Make a selection:'
+    options:
+      - value: 'non-web-yes'
+        displayName: 'Yes'
+      - value: 'non-web-no'
+        displayName: 'No'
+
+  - optionType: python-docker
+    label: "We'll give you some basic tips for docker-run"
```

Comment:
This feels like an incomplete sentence.

Reply:
@paperclypse "docker-run" is a way to execute Docker, as opposed to docker-compose. I could modify this, if you think that might help: "We'll give you some basic tips for how to set up the agent using the docker-run command."

Reply:
This is an example that uses docker-run as opposed to docker-compose. Alas, now that I know how to make these labels appear as empty, I'll just remove it.